stuffioAPI/stuffioAPI/Models/User.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace stuffioAPI.Models

```

Comment:
This is not  in the database. And never will be. Put in different folder/namespace. `ViewModels` is not a bad choice.